src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -17,6 +17,7 @@
     <!-- SYSTEM_NET_PRIMITIVES_DLL is required to allow source-level code sharing for types defined within the
          System.Net.Internals namespace. -->
     <DefineConstants>$(DefineConstants);SYSTEM_NET_PRIMITIVES_DLL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
This seems to work 
 <DefineConstants Condition="'$(Configuration)' == 'uap-Windows_NT-Debug'">$(DefineConstants);uapaot</DefineConstants>
    <DefineConstants Condition="'$(Configuration)' == 'uap-Windows_NT-Release'">$(DefineConstants);uapaot</DefineConstants>